.gitignore

```diff
@@ -27,4 +27,4 @@ coverage
 *.sln
 *.sw?
 
-*.tsbuildinfo
```

Comment:
why do we change this?? I mean what makes you remove it or why it cannot be on the file?